pyEpiabm/pyEpiabm/tests/test_utility/test_transition_time_matrix.py

```diff
@@ -0,0 +1,169 @@
+                                                        transition_matrix,
+                                                        new_transition_time)
+        self.assertEqual(10.0,
+                         transition_matrix.loc['Susceptible', 'Exposed'])
+
+        # Test error for incorrect columns is raised
+        class TestInfectionStatus(Enum):
+            Susceptiblesssss = 1
+
+        with self.assertRaises(ValueError):
+            row = TestInfectionStatus.Susceptiblesssss
+            column = TestInfectionStatus.Susceptiblesssss
+            matrix_object.update_transition_time_with_float(row, column,
+                                                            transition_matrix,
+                                                            1.0)
+
+        with self.assertRaises(ValueError):
+            row = None
+            column = None
+            matrix_object.update_transition_time_with_float(row, column,
+                                                            transition_matrix,
+                                                            1.0)
+
+        # Test error for incorrect transition time is raised
+        with self.assertRaises(ValueError):
+            row = InfectionStatus.Susceptible
+            column = InfectionStatus.Susceptible
+            matrix_object.update_transition_time_with_float(row, column,
+                                                            transition_matrix,
+                                                            -5.0)
+
+    def test_update_transition_time_with_icdf(self):
+        # Test method updates transition time as expected
+        matrix_object = TransitionTimeMatrix()
+        row_status = InfectionStatus.Susceptible
+        column_status = InfectionStatus.Exposed
+        transition_time_icdf = np.ones(10)
+        transition_time_icdf_mean = 10.0
+        transition_matrix = matrix_object.create_transition_time_matrix()
+        matrix_object.update_transition_time_with_icdf(
+            row_status,
+            column_status,
+            transition_matrix,
+            transition_time_icdf,
+            transition_time_icdf_mean)
+        test_updated_icdf = transition_matrix.loc['Susceptible', 'Exposed']
+        self.assertEqual(10.0,
+                         test_updated_icdf.mean)
+        self.assertEqual((np.ones(10)).all(),
+                         test_updated_icdf.icdf_array.all())
+
+        # Test error for incorrect columns is raised
+        class TestInfectionStatus(Enum):
+            Susceptiblesssss = 1
+
+        with self.assertRaises(ValueError):
+            row = TestInfectionStatus.Susceptiblesssss
+            column = TestInfectionStatus.Susceptiblesssss
+            test_mean = 1.0
+            test_icdf = np.ones(10)
+            matrix_object.update_transition_time_with_icdf(
+                row,
+                column,
+                transition_matrix,
+                test_icdf,
+                test_mean)
```

Comment:
```suggestion
                row, column, transition_matrix,
                test_icdf, test_mean)
```
I don't like the really long function calls, fight me (or change the others too ;) )